node/node.go

```diff
@@ -1170,6 +1170,7 @@ func (n *Node) initVMs() error {
 	}
 
 	durangoTime := version.GetDurangoTime(n.Config.NetworkID)
+	eForkTime := version.GetEForkTime(n.Config.NetworkID)
```

Comment:
Can we use "upgrade" instead of "fork"? I think we generally avoid "fork"

Reply:
Done